x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/saved_object/engine_descriptor_type.ts

```diff
@@ -30,6 +30,9 @@ export const entityEngineDescriptorTypeMappings: SavedObjectsType['mappings'] =
       type: 'integer',
       index: false,
     },
+    error: {
+      type: 'object',
+    },
```

Comment:
Are we searching over this value? If not then we don't need to include it in mappings.

Reply:
so I dont need to add a migration for it? I can just "silently" add it to the saved object?

Reply:
I ended up removing the migration, seems like that is the correct way to do this since we never want to search on errors